<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opisWykresu</class>
 <widget class="QWidget" name="opisWykresu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>95</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="text">
       <string>Zapisz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OpenBtn">
      <property name="text">
       <string>Otw√≥rz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="sourceLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectBtn">
      <property name="text">
       <string>Wybierz plik</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="destLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="CopyBtn">
      <property name="text">
       <string>Kopiuj plik</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
